charts/redpanda/Chart.yaml

```diff
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 4.0.0
+version: 4.0.1
```

Comment:
nit: this could be a separate commit

Reply:
We like to keep them together. It's handy for `git log -S`.